src/ui/MessageContent/MessageBody/index.tsx

```diff
@@ -63,7 +63,7 @@ export default function MessageBody(props: MessageBodyProps): ReactElement {
   const statefulFileInfoList = useFileInfoListWithUploaded(message); // For MultipleFilesMessage.
 
   const messageTypes = getUIKitMessageTypes();
-  const isOgMessageEnabledInGroupChannel = channel?.isGroupChannel() && config.groupChannel.enableOgtag;
+  const isOgMessageEnabledInGroupChannel = channel?.isGroupChannel() && config?.groupChannel.enableOgtag;
```

Comment:
Seems we don't allow `undefined` for `config`. If there's possible it can be, we'd better change its type or better to remove the optional chaining here :)

Reply:
You are right. Let me remove it.